checkPass.js

```diff
@@ -0,0 +1,32 @@
+
+let users = [{ email: 'pugster@malone', password: 'cheddarcheese', loginAttempts: 0 }, { email: 'thedame@tails', password: 'pawicure', loginAttempts: 0 }, { email: 'ratty@tails', password: 'ratchaser', loginAttempts: 0 }]
+
+
+authenticate = (req, res, next) => {
+
+    let currentuser = users.filter(user => user.email === req.body.email)[0] 
+    
+    let index = users.findIndex(user => user.email === req.body.email)
+    console.log(users[index].loginAttempts)
+
```

Comment:
You could use `index` to pull out the`currentuser`, save you doing another filter.
e.g. `currentUser = users[index]`